requirements.txt

```diff
@@ -1 +0,0 @@
-requirements.pip
```

Comment:
Do we need to copy the packages from requirements.pip to here?

Reply:
hmmm... I thought I fixed this in the script. I'll look into it.
Good catch!